### PR TITLE
fixed front and back issues for a signUp form

### PR DESCRIPTION
**Pull request by rostom44**
Pour ce pull request, c'est une correction de backend et front

### CODE REVIEW

**Review by MaksimLsk — APPROVED** [review]
good

**Comment by benrcd** [comment]
pending conflicts

File: client/src/pages/SignUp.jsx [hunk 1]
```diff
@@ -210,9 +238,14 @@ function SignUp() {
         <button className="button" id="signupbut" type="submit">
           <h3>Submit</h3>
```
Comment (benrcd) [inline client/src/pages/SignUp.jsx]: You know what to do to this `h3` tag

**Review by benrcd — CHANGES_REQUESTED** [review]

File: client/src/pages/SignUp.jsx (hunk 1 above)
Comment (ROS-44) [reply client/src/pages/SignUp.jsx]: 🙃 yeah

**Review by HazeFury — APPROVED** [review]
ok

**Pull request merged**